ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -566,115 +554,79 @@ def get_disk_safety(logger):
+            total_objects = disk_safety_info['total_objects']
+            # limit to 4 numbers
+            repair_percentage = float('{0:.4f}'.format((float(objects_to_repair) / total_objects) * 100)) if total_objects != 0 else 0
+            if disk_lost == 0:
+                logger.success('Backend {0} has no disks that are lost.'.format(backend_name))
+            else:
+                msg = "Another {0} disks can be lost." if lowest_safety > 0 else "Losing more disks will cause data loss!"
+                logger.failure('Backend {0} has lost {1} disk(s). {2}'.format(backend_name, disk_lost, msg))
+
+            msg = 'Backend {0}: {1} out of {2} objects have to be repaired.'.format(backend_name, objects_to_repair, total_objects)
+            if objects_to_repair == 0:
+                logger.success(msg)
+            else:
+                logger.warning(msg)
+            if objects_no_safety > 0:
+                logger.failure('Backend {0}: {1} out of {2} objects will be beyond repair if another disk fails.')
+            if objects_lost > 0:
+                logger.failure('Backend {0}: {1} out of {2} objects are beyond repair.')
+            # logger.warning('Backend {0}: {1}% of the objects have to be repaired'.format(backend_name,
+            #                                                                              repair_percentage))
+            # Log if the amount is rising
+            cache = CacheHelper.get()
+            repair_rising = False
+            if cache is None or cache.get(backend_name, None) is None:
+                # First run of healthcheck
+                logger.success('Object repair for backend_name {0} will be monitored on incrementations.'.format(backend_name))
+            elif objects_to_repair == 0:
+                # Amount of objects to repair are rising
+                logger.success('No objects in objects repair queue for backend_name {0}.'.format(backend_name))
+            elif cache[backend_name]['object_to_repair'] > objects_to_repair:
+                # Amount of objects to repair is descending
+                logger.success('Amount of objects to repair is descending for backend_name {0}.'.format(backend_name))
+            elif cache[backend_name]['object_to_repair'] < objects_to_repair:
+                # Amount of objects to repair is rising
+                repair_rising = True
+                logger.failure('Amount of objects to repair are rising for backend_name {0}.'.format(backend_name))
+            elif cache[backend_name]['object_to_repair'] == objects_to_repair:
+                # Amount of objects to repair is the same
+                logger.success('Amount of objects to repair are the same for backend_name {0}.'.format(backend_name))
+
+            if logger.print_progress is False:
+                # Custom recap for operations
+                logger.custom(
```

Comment:
Now this is completely unreadable :)
Please form your string first and then format, not mixed
Eg:
```
logger.custom('line 1 of my super long information message'
                        'line 2 of my super long information message'
                        '{0}_{1}'
                        'line 3 of my super long information message'
                        .format(<item1>, <item2>))
```